lineman/app/components/thread_page/activity_card/new_comment.haml

```diff
@@ -17,9 +17,7 @@
       %h3.new-comment__in-reply-to{ng-if: 'comment.parentId', id: 'event-{{event.id}}'}
         %strong
           %a{lmo-href-for: 'comment.author()'}> {{ comment.authorName() }}
-        %span{translate: 'new_comment_item.in_reply_to'}
-        %strong
-          %a{lmo-href-for: 'comment.parentAuthor()'}> {{ comment.parentAuthorName }}
+        %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient: '{{comment.parentAuthorName}}'}
```

Comment:
Oh, I see. specifically the case where we're jumping directly to a comment
and not back-loading it's parent.

Should we be serializing out the parentAuthorUsername alongside the
parentAuthorName then?

On Fri, Dec 18, 2015 at 3:11 AM, Robert Guthrie <notifications@github.com>
wrote:

> In lineman/app/components/thread_page/activity_card/new_comment.haml
> <https://github.com/loomio/loomio/pull/2836#discussion_r47987588>:
>
> > @@ -17,9 +17,7 @@
> >        %h3.new-comment__in-reply-to{ng-if: 'comment.parentId', id: 'event-{{event.id}}'}
> >          %strong
> >            %a{lmo-href-for: 'comment.author()'}> {{ comment.authorName() }}
> > -        %span{translate: 'new_comment_item.in_reply_to'}
> > -        %strong
> > -          %a{lmo-href-for: 'comment.parentAuthor()'}> {{ comment.parentAuthorName }}
> > +        %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient: '{{comment.parentAuthorName}}'}
>
> I don't understand what you're saying.
>
> We don't serialize the parent author with the comment, so there are cases
> when the parent author is not loaded in the client so we don't know their
> user id
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2836/files#r47987588>.
>